### ISSUE TITLE
Is there a way to have hard limits for cells (ie no word wrapping)

### ISSUE THREAD

**Issue by EliteTechnicalCare** [techprimate/TPPDF#158]
Is there any way to disable word wrapping, and either making it so it has ..., reduce text size or nothing at all.

**Comment by techprimate-phil** [comment]
Basically limiting the height of a cell an truncating the text?

**Comment by EliteTechnicalCare** [comment]
Yes i knew there was a word for it, does such options exist presently?

**Comment by techprimate-phil** [comment]
No it does not exist yet, but this sounds like a useful setting, as I am working on an improved table algorithm anyways (#41) therefore I will convert this issue in a feature request

**Comment by 96kaitlinburke** [comment]
What's the progress on this? I need to have a table with square cells, and it doesn't seem like there's currently a way to do this since the cell heights can't be set.

**Comment by philprime** [comment]
As of now, the new table logic is pre-release/released in `2.0.0`.
Therefore this is where we could start of implementing this feature

**Comment by marioradonic** [comment]
@philprime Is this something you're still interested in? I need something similar like this, so I've started working on a solution, if you want I can create a draft PR with it. Basically, I've added the `heights` property to the table similar to `widths`, so that you can set relative row heights for a full-page table.

**Comment by philprime** [comment]
I am always happy to review PRs :)

**Comment by theolampert** [comment]
@philprime @marioradonic Did you two make it any further on this feature? Is there an example or branch somewhere I can take a look at? We also need this functionality so I'd be happy to contribute to this feature if either of you pointed me in the right direction.

**Comment by philprime** [comment]
Hi @theolampert there hasn't been any progress from my side, but still I am happy to review PRs or guide any contribution :)

**Comment by theolampert** [comment]
@philprime Looking into the problem a bit more today I would actually need to support fixed width and height cells for a specific label printing spec rather than relative heights and widths. I'm thinking that TPPDF might not the right library to try to do this and rather implement the functionality with Core Graphics.

**Comment by CodeSales** [comment]
Hello, how is this issue going?

**Comment by brendand** [comment]
I would also love this feature. Could the code from @marioradonic be merged in with master?

**Comment by philprime** [comment]
I am happy to review any proposed solutions, and even using existing code ideas to implement this. Where can I find the progress of @marioradonic?

**Comment by brendand** [comment]
> I am happy to review any proposed solutions, and even using existing code ideas to implement this. Where can I find the progress of @marioradonic?

I found it a bit higher up in the thread:

https://github.com/marioradonic/TPPDF/commit/19a13793efbe80048782ce157cb78f7d0c6fcdc2
